pkg/sql/upsert.go

```diff
@@ -34,13 +34,14 @@ import (
 var upsertExcludedTable = parser.TableName{TableName: "excluded"}
 
 type upsertHelper struct {
-	p                  *planner
-	evalExprs          []parser.TypedExpr
-	sourceInfo         *dataSourceInfo
-	excludedSourceInfo *dataSourceInfo
-	allExprsIdentity   bool
-	curSourceRow       parser.Datums
-	curExcludedRow     parser.Datums
+	p                      *planner
+	evalExprs              []parser.TypedExpr
+	sourceInfo             *dataSourceInfo
+	excludedSourceInfo     *dataSourceInfo
+	allUpdateExprsIdentity bool
```

Comment:
Can you add a comment here to remind the reader of the meaning of this new attribute (and perhaps the others as well while you're at it).